tracer/src/Datadog.Trace/Propagators/ParseUtility.cs

```diff
@@ -7,44 +7,18 @@
 
 using System;
 using System.Collections.Generic;
+using System.Diagnostics.Contracts;
 using System.Globalization;
 using Datadog.Trace.Headers;
 using Datadog.Trace.Logging;
+using Datadog.Trace.Util;
 
 namespace Datadog.Trace.Propagators
 {
     internal class ParseUtility
     {
         private static readonly IDatadogLogger Log = DatadogLogging.GetLoggerFor<ParseUtility>();
 
-#if NETCOREAPP
-        public static ulong ParseFromHexOrDefault(ReadOnlySpan<char> value)
```

Comment:
Note to reviewers: this method was the odd one out for not using `<TCarrier, TCarrierGetter>` or dealing with multiple values. All it did was parse a hex string into a `ulong`.